plotly/offline/offline.py

```diff
@@ -171,6 +184,34 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
                            height=height, width=width)))
 
 
+def iplot_mpl(mpl_fig,mpl_to_plotly_kw={},iplot_kw={}):
```

Comment:
+ spacing between the args
`def iplot_mpl(mpl_fig, mpl_to_plotly_kw={}, iplot_kw={}):`